### PR TITLE
Cour 602

### PR DESCRIPTION
**Pull request by MTSarwar**

### CODE REVIEW

File: src/Dfc.CourseDirectory.Web/ViewComponents/Apprenticeships/ChooseLocation/Default.cshtml [hunk 1]
```diff
@@ -29,7 +29,29 @@
 
 <script>
     (function ($) {
-        $(function () {
+        $(function ()
```
Comment (gunndabad) [inline src/Dfc.CourseDirectory.Web/ViewComponents/Apprenticeships/ChooseLocation/Default.cshtml]: Does this need to be an AJAX request? Could maybe use formaction on the button and have it go direct to the action.

File: src/Dfc.CourseDirectory.Web/Controllers/ApprenticeshipsController.cs [hunk 2]
```diff
@@ -1378,6 +1378,18 @@ public async Task<IActionResult> DeleteDeliveryOption(DeleteDeliveryOptionViewMo
             return RedirectToAction(model.Combined ? "DeliveryOptionsCombined" : "DeliveryOptions", "Apprenticeships", new { message = "Location " + model.LocationName + " deleted", mode = model.Mode });
         }
 
+        [Authorize]
+        //public IActionResult AddNewVenue(Guid[] projectId)
```
[the remark this answers is not in the document]
Comment (MTSarwar) [reply src/Dfc.CourseDirectory.Web/Controllers/ApprenticeshipsController.cs]: removed

File: src/Dfc.CourseDirectory.Web/Controllers/ApprenticeshipsController.cs [hunk 3]
```diff
@@ -1378,6 +1378,18 @@ public async Task<IActionResult> DeleteDeliveryOption(DeleteDeliveryOptionViewMo
...
+        public IActionResult AddNewVenue(AddCourseRequestModel model)
+        {
+            _session.SetString("Option", "AddNewVenueForApprenticeships");
+            var DeliveryOptionsViewModel = _session.GetObject<DeliveryOptionsViewModel>("DeliveryOptionsViewModel");
```
[the remark this answers is not in the document]
Comment (MTSarwar) [reply src/Dfc.CourseDirectory.Web/Controllers/ApprenticeshipsController.cs]: Yes, it requried to remember in session for call back.

File: src/Dfc.CourseDirectory.Web/ViewComponents/Apprenticeships/ChooseLocation/Default.cshtml (hunk 1 above)
Comment (MTSarwar) [reply src/Dfc.CourseDirectory.Web/ViewComponents/Apprenticeships/ChooseLocation/Default.cshtml]: Yes, with current flow, some how direct call on button was not navigating

**Pull request merged**